app/service.js

```diff
@@ -27,7 +27,15 @@ function getDocumentMetadata(url, window) {
     metadata.favicon_url = parsedUrl.protocol + '//' + parsedUrl.host + '/favicon.ico';
   }
 
-  metadata.image = metadata.image && metadata.image.replace(/^\/\//, 'https://');
+  metadata.image_url = metadata.image_url && metadata.image_url.replace(/^\/\//, 'https://');
+  metadata.images = [{
+    'url': metadata.image_url,
```

Comment:
Actually, re-looking at this, it looks funny to me.

For example, on L30, we say `metadata.image_url && ...`, but here on L31-32, we blindly set the `images[]` array to `metadata.image_url` and then return a `width`, `height`, `url`, and `entropy`.

What happens if `metadata.image_url` is not truthy?
We'd return `url: undefined` and still a `width: 500`, `height: 500` and `entropy: 1`.

Not sure if we should wrap everything in `if (metadata.image_url) { ... }` so we don't return an `images[]` array (or return an empty array) if `metadata.image_url` is falsey.